### PR TITLE
docs: :memo: add `alen-rubilar.md`

### PR DESCRIPTION
**Pull request by 000alen**

### CODE REVIEW

File: 2024/fall/Scheffler-MW@11am/alen-rubilar.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Hello, there!
+
+My name is **Alen Rubilar**, I'm an M26 from Chile double majoring in ~~I don't really know what I'm doing~~ Computer Science and Natural Science
```
Comment (CarlKho-Minerva) [inline 2024/fall/Scheffler-MW@11am/alen-rubilar.md:3]: uhm akshually?

Comment (CarlKho-Minerva) [reply 2024/fall/Scheffler-MW@11am/alen-rubilar.md:3]: cool beans 😎

**Review by jonathanshamwana — COMMENTED** [review]
Ooo, nice use of squiggly lines.

File: 2024/fall/Scheffler-MW@11am/alen-rubilar.md (hunk 1 above)
Comment (CarlKho-Minerva) [reply 2024/fall/Scheffler-MW@11am/alen-rubilar.md:3]: idk what ure doing is whimsical enough eh? then again I don't doubt ur md skills so APPROVED
image

**Review by CarlKho-Minerva — APPROVED** [review]
image

**Pull request merged**